src/transfers.rs

```diff
@@ -25,6 +25,10 @@ use typed_index_collection::{Collection, CollectionWithId, Idx};
 
 type TransferMap = HashMap<(Idx<StopPoint>, Idx<StopPoint>), Transfer>;
 
+/// The closure that will determine whether a connection should be created between 2 stops.
+/// See See [generates_transfers](./fn.generates_transfers.html).
+pub type NeedTransfer<'a> = Box<dyn 'a + FnMut(&Model, Idx<StopPoint>, Idx<StopPoint>) -> bool>;
```

Comment:
Why allowing mutation of the context?
```suggestion
pub type NeedTransfer<'a> = Box<dyn 'a + Fn(&Model, Idx<StopPoint>, Idx<StopPoint>) -> bool>;
```

Reply:
It was necassary until I change my code in tartare-tools

Reply:
done